### PR TITLE
Added units for labels in launch_morphometrics_computation

### LINKED ISSUE #568: Display morphometrics units in CSV
As suggested by @jcohenadad, it would be helpful to display the units of every metric in the csv output.

e.g. something like this:

Screenshot_20210923_131711

### PR DESCRIPTION
**Pull request by michaelrizvi**
Resolves #568

### CODE REVIEW

**Review by jcohenadad — DISMISSED** [review]
LGTM

**Comment by mariehbourget** [comment]
Hi @michaelrizvi!
Have you tried outputting the morphometrics qith the FSLeyes plugin?
Unfortunately I don't have time to test it now but there may be changes to do also [here](https://github.com/neuropoly/axondeepseg/blob/5596b728b37f76ad2b83cbec3fdbf9cc7e2bfce8/ads_plugin.py#L672) and [here](https://github.com/neuropoly/axondeepseg/blob/5596b728b37f76ad2b83cbec3fdbf9cc7e2bfce8/ads_plugin.py#L696).

**Comment by Stoyan-I-A** [comment]
> Have you tried outputting the morphometrics with the FSLeyes plugin?

To be fair, we should refactor this in order to avoid having repeated code.

**Comment by michaelrizvi** [comment]
Added the units in the ads_plugin.py and tested. Everything seems to work fine.

**Comment by jcohenadad** [comment]
> To be fair, we should refactor this in order to avoid having repeated code.

@michaelrizvi could you please address here or create a specific issue?

also there is a conflict:
image

**Comment by michaelrizvi** [comment]
> > To be fair, we should refactor this in order to avoid having repeated code.
> 
> @michaelrizvi could you please address here or create a specific issue?
> 
> also there is a conflict: image

Thanks for the feedback! I have resolved the conflict, created a new issue and assigned myself to the issue

**Comment by jcohenadad** [comment]
> Thanks for the feedback! I have resolved the conflict, created a new issue and assigned myself to the issue

thank! in general it is a good habit to always link the issue you created for easier cross-ref

**Comment by jcohenadad** [comment]
> thank! in general it is a good habit to always link the issue you created for easier cross-ref

can you please do it?

**Comment by michaelrizvi** [comment]
> > thank! in general it is a good habit to always link the issue you created for easier cross-ref
> 
> can you please do it?

Both issues should now be linked to the Pull Request

**Comment by jcohenadad** [comment]
> Edit: I believe you meant linked in the comment, I did that too

yup! that's what i meant 👍

File: ads_plugin.py [hunk 1]
```diff
@@ -687,19 +687,19 @@ def on_compute_morphometrics_button(self, event):
...
+                        ('axon_area (um\u00b2)', 'f4'), # unicode for ^2
+                        ('axon_perimeter (um)', 'f4'),
+                        ('myelin_area (um\u00b2)', 'f4'),
+                        ('axon_diam (um)', 'f4'),
+                        ('myelin_thickness (um)', 'f4'),
+                        ('axonmyelin_area (um\u00b2)', 'f4'),
+                        ('axonmyelin_perimeter (um)', 'f4'),
+                        ('solidity', 'f4'),
+                        ('eccentricity', 'f4'),
+                        ('orientation', 'f4')
```
Comment (hermancollin) [inline ads_plugin.py:702]: This part is wrong.

**Comment by hermancollin** [comment]
Aside from a small bug, this PR was left in the cold but did work. It now also works with the plugin.

**Comment by mathieuboudreau** [comment]
> Aside from a small bug, this PR was left in the cold but did work. It now also works with the plugin.

If you think it's ready, please re-approve!

**Review by hermancollin — APPROVED** [review]

**Pull request merged**